test/libsolidity/syntaxTests/errors/weird_name.sol

```diff
@@ -1,8 +0,0 @@
-error error(uint a);
```

Comment:
Does it make sense to keep this test?

Reply:
Maybe. I think @hrkrshnn added a syntax test only because the semantic one got removed. It's pretty much identical to `semanticTests/errors/weird_name.sol`. But I can add it back if you think we should keep both.

Reply:
Ah right!